packages/front-end/pages/getstarted.tsx

```diff
@@ -34,18 +38,32 @@ const GetStartedPage = (): React.ReactElement => {
     return <LoadingOverlay />;
   }
 
-  return (
-    <>
-      <div className="container pagecontents position-relative">
-        <GetStarted
-          experiments={experiments?.experiments || []}
-          features={features}
-          mutateExperiments={mutateExperiments}
-          onboardingType={null}
-        />
-      </div>
-    </>
-  );
+  if (guidedOnboarding) {
```

Comment:
The new guided onboarding only works for admins (the old page has lots of logic to show/hide CTAs depending on permissions).  To keep things simple, lets just only show the new page if the person has admin permissions.  We could control this rollout via the feature flag, but I think it's safer to put the check directly in the code in this case.

```ts
const permissions = usePermissions();

if (permissions.organizationSettings && guidedOnboarding) {
```